get_csv.sh

```diff
@@ -1,2 +1,5 @@
-wget https://cernbox.cern.ch/index.php/s/SAHcpBDTp4P9jfO/download
-tar -xzf download
\ No newline at end of file
+data_path=${1:-'.'}
+mkdir -p $data_path
+
+wget -P $data_path https://cernbox.cern.ch/index.php/s/SAHcpBDTp4P9jfO/download
+tar -xzf $data_path/download
```

Comment:
if you modify this script you should probably add changes to README.md

Reply:
No changes are needed to reproduce experiments described in paper.